test/integration/full/contrast/code-highlighting.js

```diff
@@ -1,5 +1,18 @@
 describe('color-contrast code highlighting test', function() {
 	'use strict';
+	var results;
+
+	before(function(done) {
```

Comment:
This will undo the changes I made to fix the flaky test. The before that waits for the window load event needs to happen first before we run axe on the file.